internal/kuberesource/parts.go

```diff
@@ -39,9 +39,6 @@ func ContrastRuntimeClass(platform platforms.Platform) (*RuntimeClassConfig, err
 // NodeInstallerConfig wraps a DaemonSetApplyConfiguration for a node installer.
 type NodeInstallerConfig struct {
 	*applyappsv1.DaemonSetApplyConfiguration
-	*applycorev1.ServiceAccountApplyConfiguration
-	*applyrbacv1.ClusterRoleApplyConfiguration
-	*applyrbacv1.ClusterRoleBindingApplyConfiguration
```

Comment:
Can we also remove the NodeInstallerConfig as wrapper type and just use the ApplyConfig?